README.md

```diff
@@ -20,7 +20,7 @@
 
 # Hamilton
 
-The general purpose micro-orchestration framework for creating [dataflows](https://en.wikipedia.org/wiki/Dataflow) from python functions! That is, your single tool to express things like data, ML, LLM pipelines/workflows, and even web request logic!
+The general purpose micro-orchestration framework for building [dataflows](https://en.wikipedia.org/wiki/Dataflow) from python functions. Express data, ML, LLM pipelines/workflows, and web requests with a clear, easy-to-follow API.
```

Comment:
functions > API.

Reply:
yes, we mention functions in the prior sentence

Reply:
API is a confusing term for me here.

Reply:
Calling it "interface"